Radiant/fileWatcher.js

```diff
@@ -170,6 +171,14 @@ const uploadFile = function (info, endStream){
                         }, process.env.TIMEOUT_TO_CREATE_VIDEO_OBJECT);
                     }
 
+                    const seg = segment.substr(1,segment.length);
+                    const seggy = seg.split('.')[0];
```

Comment:
seg and seggy? 🧐

Reply:
you are just jealous of my naming conventions

Reply:
I have no idea what seggy meany